### ISSUE TITLE
keybindings not working for float toggling

### ISSUE THREAD

**Issue by hallyn** [0intro/wmii#3]
I have:
```
Key $MODKEY-space   # Toggle between floating and managed layers
        wmiir xwrite /tag/sel/ctl select toggle

KeyGroup Moving clients around
Key $MODKEY-Shift-$LEFT   # Move selected client to the left
        wmiir xwrite /tag/sel/ctl send sel left
Key $MODKEY-Shift-$RIGHT  # Move selected client to the right
        wmiir xwrite /tag/sel/ctl send sel right
Key $MODKEY-Shift-$UP     # Move selected client up
        wmiir xwrite /tag/sel/ctl send sel up
Key $MODKEY-Shift-$DOWN   # Move selected client down
        wmiir xwrite /tag/sel/ctl send sel down

Key $MODKEY-Shift-space   # Toggle selected client between floating and managed layers
        wmiir xwrite /tag/sel/ctl send sel toggle
```

If, from a terminal, i simply type "wmiir xwrte /tag/sel/ctl send sel toggle", that works.  But typing mod-shift-space does not work, nor does mod-space for switching between the float and non-float layers.

Any hints on where to look?

**Comment by schaiba** [comment]
Shouldn't that be "Space" instead of "space" ? Note the capitalization.

**Comment by hallyn** [comment]
I had wondered that and looked at other .wmiirc samples online, none of which capitlized.  Also the manpage shows

        Mod-Shift-space     Toggle the current window between the managed and floating layer

In any case I tried it just now, did not help.  So then I thought to check the gnome shortcuts, and saw these were in fact mapped with gnome.  Removed those, but still no good (with either space or Space).  I suspect something else is still intercepting it and it's not wmii's doing...  So for now I'm using shift-b and shift-n.

I'll close this as I suspect it's something else interfering.  Thanks.
